libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -60,13 +81,108 @@ export const AccountsManager = ({ partyId }: AccountsManagerProps) => {
     },
     [gridRef]
   );
+
   const { data, error, loading } = useDataProvider<
     AccountFieldsFragment[],
     AccountFieldsFragment
   >({ dataProvider: accountsDataProvider, update, variables });
+  const [depositDialog, setDepositDialog] = useState(false);
+
+  const symbols = data && getSymbols(data);
+  return (
+    <Web3Container>
+      <AsyncRenderer loading={loading} error={error} data={assetSymbols}>
+        {symbols &&
+          symbols.map((assetSymbol, i) => (
+            <AssetAccountTable
```

Comment:
question (non blocking): I really don't like this approach of rendering multiple data grids for each asset. I know we do a similar thing in the positions table but IMO this is the wrong approach. Lets discuss